Dockerfile

```diff
@@ -147,8 +147,13 @@ RUN git clone https://github.com/genome/bam-readcount.git /tmp/bam-readcount-0.7
 ##########
 #fpfilter#
 ##########
+WORKDIR /opt
+wget --no-check-certificate https://raw.githubusercontent.com/genome/fpfilter-tool/v0.1.0/fpfilter.pl
 COPY fpfilter.pl /usr/bin/fpfilter.pl
 
+WORKDIR /
+rm /opt/fpfilter.pl
```

Comment:
Another `RUN` here.

(For maximum efficiency, the `wget`, move/copy, and `rm` could all be in one `RUN` to avoid extra layers.)